packages/commands/src/index.ts

```diff
@@ -151,6 +151,18 @@ export class CommandRegistry {
     this._commandChanged.emit({ id, type: id ? 'changed' : 'many-changed' });
   }
 
+  /**
+   * Get the description for a specific command.
+   *
+   * @param id - The id of the command of interest.
+   *
+   * @returns The description for the command.
+   */
+  describedBy(id: string): { args: ReadonlyJSONObject | null } {
+    let cmd = this._commands[id];
+    return cmd ? cmd.describedBy : { args: null };
```

Comment:
Should we merge this after #319 so we can use the chaining operator?

Reply:
Sure